sources/express-changes/sections/ab-schemas.adoc

```diff
@@ -3,32 +3,49 @@
 
 === General
 
-This annex provides the JSON Schema definitions for the EXPRESS Changes YAML
+This annex provides the YAML Schema definitions for the EXPRESS Changes YAML
 formats defined in this document.
 
 These schemas can be used to validate EXPRESS Changes YAML files and to
 generate documentation or tooling for working with EXPRESS change records.
 
+The schemas are defined in YAML Schema format that follows the JSON Schema
+format (draft-07).
+
+
 === Schema change schema
 
 The schema change schema defines the structure for recording modifications to
 EXPRESS schema structure across versions, as described in <<schema-change>>.
 
-The schema is defined in JSON Schema format (draft-07).
-
 [source,yaml]
 ----
 include::../schemas/schema_changes.yaml[]
 ----
 
+This schema can be used for validation by prefixing an EXPRESS Changes YAML
```

Comment:
maybe: 
This schema can be used for validation by inserting the following comment lines at the beginning of the EXPRESS changes YAML file.

Reply:
Done.